pkg/cmd/show.go

```diff
@@ -134,7 +134,7 @@ func showOperator() {
 // showUI opens the gardener landing page
 func showGardenerDashboard() {
 	showPodGarden("gardener-dashboard", "garden")
-	output, err := ExecCmdReturnOutput("bash", "-c", "export KUBECONFIG="+KUBECONFIG+"; kubectl get ingress gardener-dashboard-ingress -n garden")
+	output, err := ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "ingress", "gardener-dashboard-ingress", "-n"+getTechnicalID())
```

Comment:
I'm not sure if the namespace is correct. The gardener dashboard is usually in the garden namespace

Reply:
thanks! it's my typo and fixed